### ISSUE TITLE
Is is reasonable when the loss increases on validation set?

### ISSUE THREAD

**Issue by BenjaminChoou** [rosinality/vq-vae-2-pytorch#10]
I implement va-vaq on tensorflow. I tried vq-vae v1 (no hierarchical encoder-decoder), and the validation loss decreases. But I tried vq-vae v2 and found that validation loss increases. I don't know whether it is reasonable. I only train it about 10 epochs on my own dataset.

**Comment by BenjaminChoou** [comment]
I found the possible reason. I add some batch normalization between convolution. Maybe it is the one before quantizer causes it.
